addons/sale_order_contacts/models/res_partner.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019, Jarsa Sistemas, S.A. de C.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from odoo import api, fields, models
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    empl_id = fields.Many2many(
```

Comment:
please use a better field name also if it's a m2m field it must end with ids.

I think it's better seller_ids (check if the field exists or not in the model)